api/v1/hostdefinition_types.go

```diff
@@ -45,7 +45,7 @@ type Definition struct {
 	// +kubebuilder:validation:Optional
 	NodeNameOnStorage string `json:"nodeNameOnStorage"`
 	// +kubebuilder:validation:Optional
-	IOGroups []string `json:"ioGroups"`
+	IOGroups []int `json:"ioGroups"`
```

Comment:
why it's not considered supported as a string?

Reply:
because it contains `io_group's` ids, and they are integers